### PR TITLE
Wip master

### PR DESCRIPTION
**Pull request by azat-co**
CC @AnnJohn @vicfriedman

### CODE REVIEW

**Comment by vicfriedman** [comment]
tests erroring 
```
node-debugger-lab/test/index.js:1
on (exports, require, module, __filename, __dirname) { var output = `Account b
SyntaxError: Unexpected token ILLEGAL
```

File: README.md [hunk 1]
```diff
@@ -1 +1,35 @@
...
+## Objectives
+
+1. Use Node built-in debugger by putting statements into a given script to find a problem by using debugger
+2. Launch debugger
+3. Fix the script
+
+
+
+## Introduction
+
```
Comment (victhevenot) [inline README.md:10]: this lab seems to deal with creating a bank account and reading amount in account. Add that to the intro...something like "You've been hired by XYZ bank to fix some of their account creation code. They had some bad developers in the past that left some bugs in the code. Your job is to fix all the issues and get the code functional again."

File: README.md [hunk 2]
```diff
@@ -1 +1,35 @@
...
+In this lab, you'll debug a Node script which has an issue.
+
+## Instructions
+
+1. Open `account.js` then launch it from a command line with `$ node account`.
+2. Notice some issues (there a few of them)
+3. Put some `debugger` statements in `account.js`
```
Comment (victhevenot) [inline README.md:19]: are there issues in every function? maybe give some direction as to where to start with the debugger

**Comment by azat-co** [comment]
@vicfriedman RE: https://github.com/learn-co-curriculum/node-debugger-lab/pull/1#issuecomment-179484025

I think you have an old version of Node...
Could you run `$ node -v` for me please?

**Comment by victhevenot** [comment]
@azat yepp definitely had an older version of node and npm. i tried to go back through the node instructions to see if there was an easy way to update my version of node, but that wasn't included. i ended up uninstalling node and just reinstalling from nodejs.org. is there an easier way to do that?

**Comment by victhevenot** [comment]
please make sure the tests and readme from the solution branch (merged already) make it here!

**Comment by azat-co** [comment]
updated CC @AnnJohn @vicfriedman

**Pull request merged**